### PR TITLE
Add Equipment class and test

### LINKED ISSUE #12: Implement class for a equipment

### PR DESCRIPTION
**Pull request by it-cio**
Close #12

### CODE REVIEW

File: tests/test_equipment.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    with pytest.raises(Exception) as ex:
+        Equipment([])
+    assert ex.type == ClassInitializationError
+
+
+def test_add_part():
+    part_1 = Equipment('HP LaserJet 1020')
+    part_2 = Equipment('Repair kit for LaserJet 1020')
+    part_3 = Equipment('Fuser for LaserJet 1020')
+    part_4 = Equipment('Main Motor for LaserJet 1020')
```
Comment (Lehersun) [inline tests/test_equipment.py]: Equipment can't accept other equipment as a part

File: classes/Equipment.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        else:
+            raise ClassInitializationError('Name must be string')
+
+    def get_parts_info(self):
+        parts = self.__parts
+        return parts
+
+    def add_part(self, parts):
+        if isinstance(parts, list):
+            self.__parts += parts
```
Comment (Lehersun) [inline classes/Equipment.py]: We can't check what contains a list accepted in this method. The main idea is checking if the provided list contains only objects implemented as SparePart.
Unfortinataly we have the same problem in the SparePart class. Please fix it too

Comment (it-cio) [reply classes/Equipment.py]: Fix check add_part method and test Equipment class

File: tests/test_equipment.py (hunk 1 above)
Comment (it-cio) [reply tests/test_equipment.py]: Fix

File: classes/Equipment.py (hunk 2 above)
Comment (it-cio) [reply classes/Equipment.py]: Fix

**Review by Lehersun — APPROVED** [review]

**Comment by Lehersun** [comment]
Good work

**Pull request merged**